prod-server-integration-tests/scripts/apps_manifest_test.js

```diff
@@ -0,0 +1,77 @@
+const { WebClient } = require('@slack/web-api');
+
+const options = {};
+
+const BOT_TOKEN = process.env.SLACK_BOT_TOKEN;
+const TOOLING_TOKEN = process.env.SLACK_TOOLING_TOKEN;
+
+if (process.env.SLACK_SDK_TEST_DEV_API_URL !== undefined) {
+  options.slackApiUrl = process.env.SLACK_SDK_TEST_DEV_API_URL;
+}
+
+const client = new WebClient(BOT_TOKEN, options);
```

Comment:
We should verify if a WebClient instance can work without a bot token in this scenario.

Reply:
I think the token used for `BOT_TOKEN` is irrelevant for the testing of the endpoints since they rely on the tooling token override, correct? The endpoints will not work with a non-tooling token, if that's what you mean.

Reply:
Yes, you're right. The bot token is not necessary at all. Removing it makes the code more realistic and we should demonstrate how to use the API client usually.

Reply:
> Removing it makes the code more realistic

Won't most folks using the manifest API have a `client` that's instantiated using a bot token (or user token)? Happy to remove it, but want to clarify we're on the same page.

Reply:
When it comes to a bolt app, yes it is. However, in this case, a developer using the client for the app manifest scenario may not have any bot token, right? (say, starting a new app development). Indeed, a developer may initialize the client with a bot token (as part of a larger code base or due to whatever reasons), but this assuption does not cover all use cases. The e2e test code should make sure if it works for most common use cases. Regarding the APIs we're working on now, I believe the primary one is without a bot token, and then secoundary could be with a bot token. Does this make sense to you?

Reply:
Just approved. Once this integration test is improved, go ahead with merging this PR 👍